### DISCUSSION TITLE
Can the repo-cache-expiration duration be setup for a particular argocd application?

### DISCUSSION THREAD

**Discussion by alwaysastudent** [argoproj/argo-cd#6722] [Q&A]
We would like to not cache manifests of some applications but others. Is there a way to set repo cache expiration at the application level?

**Comment by jgwest** [comment]
Hi @alwaysastudent , not as of this moment.

**Reply by alwaysastudent** [reply]
> Hi @alwaysastudent , not as of this moment.

hi, is there any way we could customize this? or do you have any plans to add this feature?

Also, I am curious if you backport features into the 1.8.x these days?

**Reply by sbose78** [reply]
No, this isn't being planned at the moment. Even if this was done, chances of this making into 1.8 are negligible. :)
